artiq/coredevice/coredevice_generic.schema.json

```diff
@@ -511,6 +511,10 @@
                         "almazny": {
                             "type": "boolean",
                             "default": false
+                        },
```

Comment:
Maybe merge both Almaznys together and add an optional ``hw_rev`` parameter, with default being legacy to keep JSON compatibility?

Reply:
That would be pretty complicated. You'd need to define what "legacy" actually means and translate that into code. This here is straight forward and maintains compatibility.